upgrade/upgrade-scripts/execute

```diff
@@ -28,6 +30,13 @@ function usage() {
 	exit 2
 }
 
+function generate_interface_to_mac_address_map() {
+  ip -o link | grep -v "lo:" | tr -d ":" | awk '{print $2,$17}' | \
```

Comment:
This feels fragile as it relies on the specific output format of the `ip` command and the order of the fields it emits. You might consider consuming the contents of the `/sys/class/net/<ifname>/address` instead. This is a kernel interface that is unlikely to change.

Reply:
This has been fixed.